webapp/components/form/Input/Input.js

```diff
@@ -32,7 +34,8 @@ export const Input = React.forwardRef((props, ref) => {
 
   const handleValueChange = (newValue) => {
```

Comment:
this function is called by the NumberFormat input field even when the value is not changed by the user, but it changes in the passed props.
To avoid a useless (and dangerous) call to onChange, we can check that the formatted value (newValue) is different from the value coming from the props.